### PR TITLE
Creates a "Off" Preset for Air Alarms

### PR DESCRIPTION
**Pull request by Kepplerx**
## What Does This PR Do
This PR adds a System Preset for Air alarms, with all thresholds set to off. When this Preset is set, it does not affect vents or scrubbers connected to the air alarm, nor the System modes for those. If changed from "Disabled" the air alarm will cycle the air, as it would if you changed between Vox or Human.

## Why It's Good For The Game
Changing a air alarms thresholds to off, to not get constant alerts or firelocks is a pain, as there are a lot of them. This will make it easier to disable alarms, which will also make it easier for antagonists to utilise the air alarm settings, so the AI will not immidietly discover someone made a breach somewhere. Alternatively the AI could plasmaflood, with air alarms not triggering fire alerts, thereby not hindering the gas spreading. In essence this is a small quality of life change, which makes air alarms easier to use for antags mostly, but also for special engineering projects.

## Images of changes
image
The difference between setting all of these manually to -1

image
To clicking one button

## Changelog
:cl:
add: Added Disabled Air Alarm setting
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/airalarm.dm
Comment (AffectedArc07) [inline code/modules/atmospherics/machinery/airalarm.dm:202]:
```suggestion
		if(AALARM_PRESET_DISABLED)
			no_cycle_after = TRUE
			TLV = list(
```

File: code/modules/atmospherics/machinery/airalarm.dm
Comment (AffectedArc07) [inline code/modules/atmospherics/machinery/airalarm.dm:212]:
```suggestion
	if(!no_cycle_after)
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

**Comment by Carthusia** [comment]
Oh thank god, i kept having the alarms going off in my vox boxes because i couldnt get it to stop freaking out when i tried to change the alarm settings

**Comment by Kepplerx** [comment]
Tested with the new small changes, still works as intended, now just waiting for approval.

**Review by dearmochi — APPROVED** [review]

**Pull request merged**
